core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -418,39 +398,15 @@ class Invoker(
     private val activationStore = WhiskActivationStore.datastore(config)
     private val pool = new ContainerPool(config, instance, verbosity)
     private val activationCounter = new Counter() // global activation counter
-    private val userActivationCounter = new TrieMap[String, Counter]
 
     private val consul = new ConsulClient(config.consulServer)
 
-    // Restore old state from the same invoker instance
-    consul.kv.getRecurse(InvokerKeys.userActivationCount(instance)).map { partitioned =>
-        val flattened = partitioned.values.map { userCounts =>
-            userCounts.parseJson.convertTo[Map[String, Int]]
-        }.reduce(_ ++ _)
-
-        val readActivationCounter = flattened.mapValues { readCount =>
-            val cnt = new Counter()
-            cnt.set(readCount)
-            cnt
-        }
-
-        info(this, s"restored old counts from consul: $flattened")
-
-        userActivationCounter.clear()
-        userActivationCounter ++= readActivationCounter
-
-        activationCounter.set(flattened.values.sum)
-    } onComplete { _ =>
-        // Repeatedly updates the KV store as to the invoker's last check-in.
-        new ConsulKVReporter(consul, 3 seconds, 2 seconds,
+    // Repeatedly updates the KV store as to the invoker's last check-in.
+    new ConsulKVReporter(consul, 3 seconds, 2 seconds,
             InvokerKeys.hostname(instance),
             InvokerKeys.start(instance),
             InvokerKeys.status(instance),
-            { index =>
-                (if (index % 5 == 0) getUserActivationCounts() else Map[String, JsValue]()) ++
-                    Map(InvokerKeys.activationCount(instance) -> activationCounter.cur.toJson)
-            })
-    }
+            { index => Map(InvokerKeys.activationCount(instance) -> activationCounter.cur.toJson) })
```

Comment:
So it's used in global limits for the whole system.  We could either choose to do this in this PR as well (or not).